corehq/apps/integration/payments/README.md

```diff
@@ -0,0 +1,27 @@
+# MTN MoMo Payment
+The payment gateway is used to facilitate payments, typically from organizations to customers.
+
+Please follow the [Getting Started](https://momodeveloper.mtn.com/api-documentation/getting-started) section to get going.
+Note that we're interested in the **Disbursements** product.
+
+### Sandbox Environment
+In order to generate credentials for the sandbox environment see [this documentation](https://momodeveloper.mtn.com/api-details#api=sandbox-provisioning-api&operation=post-v1_0-apiuser).
+
+## Configuring the Payment Gateway in HQ
+In order to set up ConnectionSettings for the payment gateway, you will need to provide the following information
+assuming you have already created an account with the payment gateway and want to set up the sandbox:
```

Comment:
>  want to set up the sandbox:

I assume a sandbox is a test setup, the connection settings are only for sandbox?

Reply:
Not quite. I added [this commit](https://github.com/dimagi/commcare-hq/pull/35966/commits/e55f22a3b54aaeccbb6724b148cfba4eec5da6be) to make clear that there's a live env as well.

Reply:
Cool,
I was confused why we are seeing sandbox here, it could be live environment as well right?

Reply:
Yes, this documentation is the same for live env as well. I'll update to make it more clear.

Reply:
See https://github.com/dimagi/commcare-hq/pull/35966/commits/ea5ca8481138e7a82117592434a903785e982e5d.